wandb/cli/cli.py

```diff
@@ -182,39 +200,45 @@ def login(key, host, cloud, relogin, anonymously):
     wandb.login(relogin=relogin, key=key, anonymous=anon_mode)
 
 
-@cli.command(context_settings=CONTEXT, help="Run a grpc server", name="grpc-server", hidden=True)
+@cli.command(
+    context_settings=CONTEXT, help="Run a grpc server", name="grpc-server", hidden=True
+)
 @display_error
 def grpc_server(project=None, entity=None):
     _ = util.get_module(
         "grpc",
         required="grpc-server requires the grpcio library, run pip install wandb[grpc]",
     )
     from wandb.server.grpc_server import main as grpc_server
+
     grpc_server()
 
 
 @cli.command(context_settings=CONTEXT, help="Run a SUPER agent", hidden=True)
 @click.option("--project", "-p", default=None, help="The project use.")
 @click.option("--entity", "-e", default=None, help="The entity to use.")
-@click.argument('agent_spec', nargs=-1)
```

Comment:
The reformatting of everything is unfortunate.  We were waiting until the old cli was no longer being maintained so we could still diff the old logic.  It should be ok now since we're so close to officially releasing, but @raubitsj will have the final say on whether we get the reformatting in now or not.

Reply:
just add wandb/cli/cli.py to pyproject.toml black format list.  no reason not to start adding more files.  ill add a bunch when it looks like there isnt too many branches outstanding